go/datas/remote_database_handlers_test.go

```diff
@@ -158,19 +158,16 @@ func TestBuildWriteValueRequest(t *testing.T) {
 	assert.Equal(len(hints), count)
 
 	outChunkChan := make(chan interface{}, 16)
-	go deserializeToChan(gr, outChunkChan)
+	chunks.DeserializeToChan(gr, outChunkChan)
```

Comment:
Won't this hang if the code being tested tries to put more than 16 items into `outChunkChan`? Obviously it _shouldn't_ do that, but it might. It feels a bit gross to rely on the chunk buffer to avoid deadlocking.

I guess my more realistic concern is that someone expands the test to write more than 16 chunks, and you could mitigate that by making the chunk buffer be the same size as `chnx`